cfgov/jinja2/v1/_includes/molecules/contact-method.html

```diff
@@ -0,0 +1,94 @@
+
+   ==========================================================================
+
+   Description:
+
+   Create a Contact Method molecule.
+   See [GHE]/flapjack/Modules-V1/wiki/Contact-Method
+
+   contact:                  An object with the following options for a contact.
+   contact.emails:           An array containing the email addresses.
+
+   contact.phones:           An array containing phone number information.
+   contact.phones.number:    A phone number.
+   contact.phones.vanity:    An associated vanity phone number.
+   contact.phones.tty:       An associated TTY/TDD number.
+
+   contact.faxes:            An object containing fax numbers.
+
+   contact.address:          An object containing address information.
+   contact.address.label:    A string containing a label for the address.
+   contact.address.title:    A string containing a address name.
+   contact.address.street:   A string containing a street address.
+   contact.address.city:     A string containing a city.
+   contact.address.state:    A string containing a state.
+   contact.address.zip_code: A string containing a zip code.
+
+   ========================================================================== #}
+
+{% macro render(contact) %}
+<div class="m-contact-method">
+    {% if contact.emails %}
+        {% set icon = 'cf-icon-email' %}
+        {% set title = 'Email' %}
+
+        <span class="cf-icon {{ icon }} list_icon"></span>
+        <span class="h5 list_text">{{ title }}</span>
+        <p class="short-desc">
+            {% for email in contact.emails %}
+                <a class="list_link" href="mailto:{{ email }}">
+                {{ email }}
+                </a><br>
+            {% endfor %}
+        </p>
+
+    {% elif contact.phones %}
+        {% set icon = 'cf-icon-phone' %}
+        {% set title = 'Phone' %}
+
+        <span class="cf-icon {{ icon }} list_icon"></span>
+        <span class="h5 list_text">{{ title }}</span>
+        <p class="short-desc">
+            {% for phone in contact.phones %}
+                {% if phone.vanity %}
+                <span>{{ format_phone(phone.vanity) }}</span> |
+                {% endif %}
+                <span>{{ format_phone(phone.number) }}</span>
+                {% if phone.tty %}
+                    <br>TTY/TDD: {{ format_phone(phone.tty) }}<br>
+                {% endif %}
+            {% endfor %}
+        </p>
+
+    {% elif contact.faxes %}
+        {% set icon = 'cf-icon-fax' %}
+        {% set title = 'Fax' %}
+
+        <span class="cf-icon {{ icon }} list_icon"></span>
+        <span class="h5 list_text">{{ title }}</span>
+        <p class="short-desc">
+            {% for fax in contact.faxes %}
+                <span>{{ format_phone(fax) }}</span><br>
+            {% endfor %}
+        </p>
+
+    {% elif contact.address %}
+        {% set icon = 'cf-icon-mail' %}
+        {% set title = contact.address.label %}
+        <span class="cf-icon {{ icon }} list_icon"></span>
```

Comment:
`_method` is just there because the molecule name is Contact Method, but we could probably update the molecule name. @duelj What do you think about breaking this into Email Contact, Phone Contact, and Address Contact molecules?